src/user.js

```diff
@@ -11,40 +16,77 @@ const getUserGeo = () => {
           lat: result.coords.latitude,
           long: result.coords.longitude
         }
-
-        localStorage.setItem('userData', JSON.stringify(data))
         return resolve(data)
       })
     }
   })
 }
 
-// Set flag if user has joined to app
+/**
+ * Set flag if user has joined to app
+ */
 const setUserHasJoined = () => {
   localStorage.setItem('userHasJoined', JSON.stringify(true))
 }
 
-// Check if user has joined to app
+/**
+ * Check if user has joined to app
+ */
 const checkHasJoined = () => {
   return !!localStorage.getItem('userHasJoined')
 }
 
-// Check geo
-const checkUserData = () => {
-  return !!localStorage.getItem('userData')
+/**
+ * Set user geo data
+ * @param {object} data
+ */
+const setUserGeoDataToStorage = data => {
+  localStorage.setItem('userGeoData', JSON.stringify(data))
+}
+
+/**
+ * Get user geo data from localstorage
+ */
+const getUserGeoDataFromStorage = () => {
+  return JSON.parse(localStorage.getItem('userGeoData'))
```

Comment:
@raulghm sigue susceptible a lanzar un error esta función. Debe estar envuelta en un try/catch o bien dejar el try/catch cuando sea llamada.

```suggestion
  try {
    return JSON.parse(localStorage.getItem('userGeoData'))
  } catch (err) {
    return null
  }
```